2022/1002xxx/GSD-2022-1002526.json

```diff
@@ -1,68 +1,5 @@
-  },
-  "OSV": {
-    "id": "GSD-2022-1002526",
-    "modified": "2022-06-28T02:20:38.695078Z",
-    "published": "2022-06-28T02:20:38.695078Z",
-    "summary": "heap buffer overflow in OpenSSL version 3.0.4",
-    "details": "In OpenSSL version 3.0.4 a heap buffer overflow exists in the AVX512 support that can be attacked via network resulting in code execution. This is reachable via four code paths: RSAZ 1024, RSAZ 512, Dual 1024 RSAZ, and Default constant-time Montgomery modular exponentiation. Please note this issue is especially serious as OpenSSL 3.0.4 contains a security fix for CVE-2022-2068 which allows arbitrary code execution. This means that all users of OpenSSL 3 are affected by a number of serious security vulnerabilities. Intel AXV-512 support is present in Intel CPUs releases including Knights Landing (Xeon Phi x200, 2016), Knights Mill (Xeon Phi x205, 2017), Skylake-SP, Skylake-X (2017), Cannon Lake (2018), Cascade Lake (2019), Cooper Lake (2020), Ice Lake (2019), Tiger Lake (2020), Rocket Lake (2021). Please note that on Alder Lake (2021) AVX-512 is not officially supported, but can be enabled on some motherboards with some BIOS versions. For AMD all Zen4 CPUs contain AVX-512 support.",
-    "affected": [
-      {
-        "package": {
-          "name": "OpenSSL",
-          "ecosystem": "GSD"
-        },
-        "versions": [
-          "3.0.4"
-        ]
-      }
-    ],
-    "references": [
-      {
-        "type": "WEB",
-        "url": "https://github.com/openssl/openssl/issues/18625"
-      },
-      {
-        "type": "WEB",
-        "url": "https://github.com/openssl/openssl/commit/10d8a109be0fe50315e4eeb0676f6571914cd47a"
-      },
-      {
-        "type": "WEB",
-        "url": "https://www.theregister.com/2022/06/27/openssl_304_memory_corruption_bug/"
-      },
-      {
-        "type": "WEB",
-        "url": "https://guidovranken.com/2022/06/27/notes-on-openssl-remote-memory-corruption/"
-      },
-      {
-        "type": "WEB",
-        "url": "https://twitter.com/GuidoVranken/status/1539687342939820032"
-      },
-      {
-        "type": "WEB",
-        "url": "https://en.wikipedia.org/wiki/AVX-512"
-      }
-    ]
-  }
+    "GSD": {
```

Comment:
Please re-add the original ID (GSD-2022-1002526).

Reply:
but it's not that gsd id anymore. it's the new one.

Reply:
There is no indication within the file that this is GSD-2022-1002526, and that it has been deprecated in favor of GSD-2022-2274.

This is GSD-2022-1002526, and it is essentially a redirect to GSD-2022-2274. We can't have IDs that don't specify their UID. That breaks everything.

Reply:
It would be good to have a sentence in the description, to preserve the history for people who do not have the repository at hand.